assets/scripts/src/choices.js

```diff
@@ -647,8 +647,6 @@ class Choices {
 
     if (shouldFlip) {
       this.containerOuter.classList.add(this.config.classNames.flippedState);
-    } else {
-      this.containerOuter.classList.remove(this.config.classNames.flippedState);
```

Comment:
That is probably better aproach anyways, but I am still not sure why to remove this class in `showDropdown`.